### PR TITLE
Add more memory logging

### PR DESCRIPTION
**Pull request by xiaoxmeng**

### CODE REVIEW

File: velox/common/caching/AsyncDataCache.cpp [hunk 1]
```diff
@@ -652,8 +652,9 @@ bool AsyncDataCache::makeSpace(
 
 void AsyncDataCache::backoff(int32_t counter) {
   size_t seed = folly::hasher<uint16_t>()(++backoffCounter_);
-  auto usec = (seed & 0xfff) * (counter & 0x1f);
-  VLOG(1) << "Backoff in allocation contention for " << usec << " us.";
+  const auto usec = (seed & 0xfff) * (counter & 0x1f);
+  LOG_EVERY_N(WARNING, 1000)
```
Comment (mbasmanova) [inline velox/common/caching/AsyncDataCache.cpp:656]: Why is this a warning and not INFO?

File: velox/common/memory/MallocAllocator.cpp [hunk 2]
```diff
@@ -333,10 +333,11 @@ bool MallocAllocator::checkConsistency() const {
...
-  return fmt::format(
-      "[allocated bytes {}, allocated pages {}, mapped pages {}]",
-      allocatedBytes_,
-      numAllocated_,
-      numMapped_);
+  std::stringstream out;
+  out << "Memory Allocator[" << kindString(kind_) << " capacity "
+      << ((capacity_ == kMaxMemory) ? "UNLIMITED" : succinctBytes(capacity_))
+      << " allocated bytes " << allocatedBytes_ << " allocated pages "
+      << numAllocated_ << " mapped pages " << numMapped_ << "]" << std::endl;
```
Comment (mbasmanova) [inline velox/common/memory/MallocAllocator.cpp:340]: Do we want to include endl here?

**Review by mbasmanova — APPROVED** [review]
Thanks.

File: velox/common/memory/MallocAllocator.cpp (hunk 2 above)
Comment (xiaoxmeng) [reply velox/common/memory/MallocAllocator.cpp:340]: Maybe not. Let me see.

**Pull request closed without merging**